R/combine_pdfs.R

```diff
@@ -24,10 +24,16 @@ combine_pdfs <- function(input_paths, output_path) {
   assertthat::assert_that(all(fs::file_exists(input_paths)))
   assertthat::assert_that(fs::dir_exists(fs::path_dir(output_path)))
 
+  # replace `~` with users home directory for PyPDF2
```

Comment:
Add a note in the function details field to not use symlinks or aliases in file paths.

Reply:
Actually switched to `fs::path_real` which should resolve all these issues.

From the function description
```
path_real: returns the canonical path, eliminating any symbolic links and the special 
references ~, ~user, ., and .., , i.e. it calls path_expand() (literally) and path_norm() 
(effectively).
```